packages/eas-cli/src/build/build.ts

```diff
@@ -87,12 +88,16 @@ export async function prepareBuildRequestForPlatformAsync<
     }
   );
 
-  await withAnalyticsAsync(async () => await builder.syncProjectConfigurationAsync(ctx), {
-    attemptEvent: BuildEvent.CONFIGURE_PROJECT_ATTEMPT,
-    successEvent: BuildEvent.CONFIGURE_PROJECT_SUCCESS,
-    failureEvent: BuildEvent.CONFIGURE_PROJECT_FAIL,
-    trackingCtx: ctx.trackingCtx,
-  });
+  await withAnalyticsAsync(
+    ctx.analyticsManager,
```

Comment:
It feels like we are passing 2 contexts for analytics (analyticsManager, trackingCtx), maybe it should be merged?
Also maybe this helper `withAnalyticsAsync` should be part of the manager interface.

Reply:
Will make this change in a follow-up. The tracking context is really just an arbitrary key-value so it's probably a misnomer rather than a context.